README.md

```diff
@@ -1,3 +1,25 @@
 ![Docker](https://github.com/nlnwa/veidemann-browser-controller/workflows/Docker/badge.svg)
 
 # veidemann-browser-controller
+
+## Test
+
+    go test ./...
+
+## Run integration test
+
+The integration test is useful to validate compatibility between the `browserless` container image and the `chromedp` library.
+
+### Using docker
+
+    go test --tags=integration -run TestSession_Fetch ./server
```

Comment:
As discussed IRL, I tried running this test on my machine, and this is the result I got:

```
go test --tags=integration -run TestSession_Fetch ./server -count=1 -test.v
2023-06-28T08:12:29+02:00 INF Setting log level to debug
2023/06/28 08:12:29 github.com/testcontainers/testcontainers-go - Connected to docker: 
  Server Version: 24.0.2
  API Version: 1.43
  Operating System: Fedora Linux 38 (Workstation Edition)
  Total Memory: 31688 MB
2023/06/28 08:12:29 Starting container id: 2d965fa73601 image: docker.io/testcontainers/ryuk:0.3.4
2023/06/28 08:12:29 Waiting for container id 2d965fa73601 image: docker.io/testcontainers/ryuk:0.3.4
2023/06/28 08:12:29 Container is ready id: 2d965fa73601 image: docker.io/testcontainers/ryuk:0.3.4
2023/06/28 08:12:29 Starting container id: ce19e40d660c image: browserless/chrome:1.33.1-puppeteer-3.0.0
2023/06/28 08:12:30 Waiting for container id ce19e40d660c image: browserless/chrome:1.33.1-puppeteer-3.0.0
2023/06/28 08:12:30 Container is ready id: ce19e40d660c image: browserless/chrome:1.33.1-puppeteer-3.0.0
2023-06-28T08:12:30+02:00 INF Connected address=:5002 service=logWriter
2023-06-28T08:12:30+02:00 INF API server listening address=:7777
=== RUN   TestSession_Fetch
=== RUN   TestSession_Fetch/elg
2023-06-28T08:12:30+02:00 DBG Start fetch eid=eid session=1 uri=http://elg.no
2023-06-28T08:12:30+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://elg.no
2023-06-28T08:12:32+02:00 DBG Found 1 outlinks. eid=eid session=1 uri=http://elg.no
2023-06-28T08:12:32+02:00 DBG Wrote 2 crawlLogs eid=eid session=1 uri=http://elg.no
2023-06-28T08:12:32+02:00 DBG Pagelog written eid=eid session=1 uri=http://elg.no
2023-06-28T08:12:32+02:00 DBG Fetch done eid=eid session=1 uri=http://elg.no
    server_test.go:220: Resource count: 2, Time: 1206
=== RUN   TestSession_Fetch/vg
2023-06-28T08:12:36+02:00 DBG Start fetch eid=eid session=1 uri=http://vg.no
2023-06-28T08:12:36+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://vg.no
2023-06-28T08:12:36+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.268 session=1 uri=http://vg.no
2023-06-28T08:12:36+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.238 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.289 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.295 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.288 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.286 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.287 session=1 uri=http://vg.no
2023-06-28T08:12:37+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=183.294 session=1 uri=http://vg.no
2023-06-28T08:12:38+02:00 DBG Found 291 outlinks. eid=eid session=1 uri=http://vg.no
2023-06-28T08:12:38+02:00 DBG Wrote 102 crawlLogs eid=eid session=1 uri=http://vg.no
2023-06-28T08:12:38+02:00 DBG Pagelog written eid=eid session=1 uri=http://vg.no
2023-06-28T08:12:38+02:00 DBG Fetch done eid=eid session=1 uri=http://vg.no
    server_test.go:220: Resource count: 102, Time: 2132
=== RUN   TestSession_Fetch/nb
2023-06-28T08:12:42+02:00 DBG Start fetch eid=eid session=1 uri=http://nb.no
2023-06-28T08:12:42+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://nb.no
2023-06-28T08:12:43+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=264.81 session=1 uri=http://nb.no
2023-06-28T08:12:43+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=264.87 session=1 uri=http://nb.no
2023-06-28T08:12:45+02:00 DBG Found 127 outlinks. eid=eid session=1 uri=http://nb.no
2023-06-28T08:12:45+02:00 DBG Wrote 48 crawlLogs eid=eid session=1 uri=http://nb.no
2023-06-28T08:12:45+02:00 DBG Pagelog written eid=eid session=1 uri=http://nb.no
2023-06-28T08:12:45+02:00 DBG Fetch done eid=eid session=1 uri=http://nb.no
    server_test.go:220: Resource count: 48, Time: 2084
=== RUN   TestSession_Fetch/fhi
2023-06-28T08:12:49+02:00 DBG Start fetch eid=eid session=1 uri=http://fhi.no
2023-06-28T08:12:49+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://fhi.no
2023-06-28T08:12:51+02:00 DBG Found 56 outlinks. eid=eid session=1 uri=http://fhi.no
2023-06-28T08:12:51+02:00 DBG Wrote 18 crawlLogs eid=eid session=1 uri=http://fhi.no
2023-06-28T08:12:51+02:00 DBG Pagelog written eid=eid session=1 uri=http://fhi.no
2023-06-28T08:12:51+02:00 DBG Fetch done eid=eid session=1 uri=http://fhi.no
    server_test.go:220: Resource count: 18, Time: 1689
=== RUN   TestSession_Fetch/db
2023-06-28T08:12:55+02:00 DBG Start fetch eid=eid session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.82 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.121 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.122 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.128 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.131 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.135 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.132 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.133 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.123 session=1 uri=http://db.no
2023-06-28T08:12:55+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=http://db.no
2023-06-28T08:12:56+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.127 session=1 uri=http://db.no
2023-06-28T08:12:56+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.126 session=1 uri=http://db.no
2023-06-28T08:12:56+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=423.134 session=1 uri=http://db.no
2023-06-28T08:12:56+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=423.242 session=1 uri=http://db.no
2023-06-28T08:12:56+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=423.257 session=1 uri=http://db.no
2023-06-28T08:12:59+02:00 DBG Found 229 outlinks. eid=eid session=1 uri=http://db.no
2023-06-28T08:12:59+02:00 DBG New request from proxy without ID: GET https://www.dagbladet.no:443/service-worker.js
2023-06-28T08:12:59+02:00 DBG Wrote 125 crawlLogs eid=eid session=1 uri=http://db.no
2023-06-28T08:12:59+02:00 DBG Pagelog written eid=eid session=1 uri=http://db.no
2023-06-28T08:12:59+02:00 DBG Fetch done eid=eid session=1 uri=http://db.no
    server_test.go:220: Resource count: 125, Time: 3959
=== RUN   TestSession_Fetch/maps
2023-06-28T08:13:03+02:00 DBG Start fetch eid=eid session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:03+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:04+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=510.52 session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:04+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=510.29 session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:04+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=510.36 session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:05+02:00 DBG Found 5 outlinks. eid=eid session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:05+02:00 DBG Wrote 23 crawlLogs eid=eid session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:05+02:00 DBG Pagelog written eid=eid session=1 uri=https://goo.gl/maps/EmpIH
2023-06-28T08:13:05+02:00 DBG Fetch done eid=eid session=1 uri=https://goo.gl/maps/EmpIH
    server_test.go:220: Resource count: 23, Time: 1830
=== RUN   TestSession_Fetch/ranano
2023-06-28T08:13:09+02:00 DBG Start fetch eid=eid session=1 uri=https://ranano.no/
2023-06-28T08:13:09+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.226 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.242 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.235 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.476 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.230 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.224 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.511 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.513 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.515 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.526 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.514 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.516 session=1 uri=https://ranano.no/
2023-06-28T08:13:11+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.527 session=1 uri=https://ranano.no/
2023-06-28T08:13:12+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=587.544 session=1 uri=https://ranano.no/
2023-06-28T08:13:12+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=587.554 session=1 uri=https://ranano.no/
2023-06-28T08:13:12+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=587.564 session=1 uri=https://ranano.no/
2023-06-28T08:13:12+02:00 DBG Loading failed: XHR: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=587.530 session=1 uri=https://ranano.no/
2023-06-28T08:13:12+02:00 DBG Loading failed: Fetch: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=587.570 session=1 uri=https://ranano.no/
2023-06-28T08:13:15+02:00 DBG Found 98 outlinks. eid=eid session=1 uri=https://ranano.no/
2023-06-28T08:13:15+02:00 DBG Wrote 191 crawlLogs eid=eid session=1 uri=https://ranano.no/
2023-06-28T08:13:15+02:00 DBG Pagelog written eid=eid session=1 uri=https://ranano.no/
2023-06-28T08:13:15+02:00 DBG Fetch done eid=eid session=1 uri=https://ranano.no/
    server_test.go:220: Resource count: 191, Time: 5280
=== RUN   TestSession_Fetch/cynergi
2023-06-28T08:13:19+02:00 DBG Start fetch eid=eid session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:19+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:19+02:00 DBG Loading failed: Other: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.6 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:19+02:00 DBG Loading failed: Other: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.7 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:19+02:00 DBG Loading failed: Other: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.9 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:19+02:00 DBG Loading failed: Other: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.8 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.44 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.49 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.34 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.23 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.39 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.148 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.35 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.147 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.50 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.45 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:20+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.24 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.40 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.161 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Fetch: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.162 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.51 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.46 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.41 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.36 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.25 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.52 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.151 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:21+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.150 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.149 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.47 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.42 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.37 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.152 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.53 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:22+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.26 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.153 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.48 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.43 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.154 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.38 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:23+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.155 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Font: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.27 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.156 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.157 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.158 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.159 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:24+02:00 DBG Loading failed: Script: net::ERR_FAILED blockedReason= canceled=false eid=eid requestId=679.160 session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:25+02:00 DBG Found 5 outlinks. eid=eid session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:25+02:00 DBG Wrote 88 crawlLogs eid=eid session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:25+02:00 DBG Pagelog written eid=eid session=1 uri=https://www.cynergi.no/
2023-06-28T08:13:25+02:00 DBG Fetch done eid=eid session=1 uri=https://www.cynergi.no/
    server_test.go:220: Resource count: 88, Time: 6460
=== RUN   TestSession_Fetch/pdf1
2023-06-28T08:13:30+02:00 DBG Start fetch eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:30+02:00 DBG Restoring 0 browser cookies eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:30+02:00 DBG Loading failed: Document: net::ERR_ABORTED blockedReason= canceled=true eid=eid requestId=119B22B806820405514DEEE200C923FA session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:31+02:00 DBG Found 0 outlinks. eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:31+02:00 DBG Wrote 1 crawlLogs eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:31+02:00 DBG Pagelog written eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
2023-06-28T08:13:31+02:00 DBG Fetch done eid=eid session=1 uri=https://www.nb.no/content/uploads/2019/04/tildelingsbrev_nasjonalbiblioteket_2019.pdf
    server_test.go:220: Resource count: 1, Time: 1225
--- PASS: TestSession_Fetch (64.60s)
    --- PASS: TestSession_Fetch/elg (5.46s)
    --- PASS: TestSession_Fetch/vg (6.43s)
    --- PASS: TestSession_Fetch/nb (6.54s)
    --- PASS: TestSession_Fetch/fhi (5.92s)
    --- PASS: TestSession_Fetch/db (8.24s)
    --- PASS: TestSession_Fetch/maps (6.08s)
    --- PASS: TestSession_Fetch/ranano (9.78s)
    --- PASS: TestSession_Fetch/cynergi (10.71s)
    --- PASS: TestSession_Fetch/pdf1 (5.44s)
PASS
2023-06-28T08:13:35+02:00 DBG Waiting for 0 remaining sessions
2023-06-28T08:13:35+02:00 DBG All sessions finished
2023-06-28T08:13:35+02:00 INF Shutting down API server
2023-06-28T08:13:35+02:00 INF gRPC connection ended. Remaining open RPCs: 0
ok  	github.com/nlnwa/veidemann-browser-controller/server	65.900s
```